lib/theme/main_theme.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_exercise/theme/colors.dart';
+import 'text_theme.dart';
+
+ThemeData mainThemeLight = ThemeData.light().copyWith(
```

Comment:
There is nothing "main" about this. There is no "non-main" either. Can you update this with `appThemeLight` or just `lightTheme`?